docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  mongodb:
+    image: mongo:3.6
```

Comment:
What is the docker-compose file used for?

Reply:
To run tests locally. We can take it out of this PR if you think it's not valuable, or if it should be introduced separately with documentation.